tests/e2e/api/transaction_arguments.test.ts

```diff
@@ -0,0 +1,390 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Account, AptosConfig, Network, Aptos, AccountAddress } from "../../../src";
+import {
+  MAX_U128_BIG_INT,
+  MAX_U16_NUMBER,
+  MAX_U256_BIG_INT,
+  MAX_U32_NUMBER,
+  MAX_U64_BIG_INT,
+  MAX_U8_NUMBER,
+} from "../../../src/bcs/consts";
+import { Bool, U128, U16, U256, U32, U64, U8 } from "../../../src/bcs/serializable/move-primitives";
+import { MoveObject, MoveOption, MoveString, MoveVector } from "../../../src/bcs/serializable/move-structs";
+import { TypeTag } from "../../../src/transactions/typeTag/typeTag";
+import { EntryFunctionArgumentTypes } from "../../../src/transactions/types";
+import {
+  SigningScheme,
+  TransactionFeePayerSignature,
+  TransactionMultiAgentSignature,
+  UserTransactionResponse,
+} from "../../../src/types";
+
+jest.setTimeout(30000);
```

Comment:
Is there a test that's actually taking 30 seconds?

Reply:
No, I think I just changed it to 30 to make sure it wouldn't fail, I think I can do like 10 or 15